pkg/factory/factory.go

```diff
@@ -158,13 +158,17 @@ func (rf *resourceFactory) GetDeviceFilter(rc *types.ResourceConfig) (interface{
 		if err := json.Unmarshal(*rc.Selectors, netDeviceSelector); err != nil {
 			return nil, fmt.Errorf("error unmarshalling NetDevice selector bytes %v", err)
 		}
+
+		glog.Infof("net device selector for resource %s is %+v", rc.ResourceName, netDeviceSelector)
```

Comment:
perhaps put this as debug ?

Reply:
There is no glog.Debug, also glog.V() is not used in the project

Reply:
I see.
As a general note perhaps it would be good to start using leveled logging to avoid clutter when not needed.

Reply:
we could start with the logs here :) 

anyway, im fine with it as is.

Reply:
This log here is duplicate with https://github.com/intel/sriov-network-device-plugin/blob/master/cmd/sriovdp/manager.go#L86.